Eggstravaganza/Assets/Scripts/PlayerScoreNetwork.cs

```diff
@@ -132,6 +160,12 @@ void IncrementPlayerScore(int amt)
         GameData.UpdatePlayerScore(m_Score.Value);
     }
 
+    void AssignHat(int index)
+    {
+        var hat = Instantiate(m_Prefabs[index]);
+        hat.transform.parent = this.transform.gameObject.transform.GetChild(3);
```

Comment:
are these prefabs already set to a specific height and so you don't need to set a position? 

also nit:
```suggestion
        Instantiate(m_Prefabs[index], transform.GetChild(3), true);
```

Reply:
yes! their height is currently mapped to a gopher standing at (0,0,0), so we might have to play around with it a little once we integrate everything